monitoring/managing-metrics-targets.adoc

```diff
@@ -0,0 +1,30 @@
+[id="managing-metrics-targets"]
+= Managing metrics targets
+include::modules/common-attributes.adoc[]
+:context: managing-metrics-targets
+
+toc::[]
+
+{product-title} Monitoring collects metrics from targeted cluster components by scraping data from exposed service endpoints. 
+
+In the Administrator perspective in the {product-title} web console, you can use the *Metrics Targets* page to view, search, and filter the endpoints that are currently targeted for scraping. 
```

Comment:
Elsewhere in your docs, you bolden **Administrator** perspective. So do so here: 

In the **Administrator** perspective, in the OpenShift Container Platform web console...

Reply:
Good catch. Fixed.